watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskFragment.java

```diff
@@ -189,12 +199,18 @@ public void onResponse(ArrayList<Task> tasks) {
                         setAllTasks(tasks);
                         mAllTaskAdapter.notifyDataSetChanged();
                         for(int i=0; i < mAllTaskAdapter.getGroupCount(); i++) mListView.expandGroup(i);
+
                     } else {
                         hideList();
                     }
                 }
-
-                if (mSwipeLayout != null) mSwipeLayout.setRefreshing(false);
+                    new CountDownTimer(2000, 1000) {
+                        public void onTick(long millisUntilFinished) {}
+                        public void onFinish() {
+                            if (mSwipeLayout != null) mSwipeLayout.setRefreshing(false);
+                            if (mNoTasks != null) mNoTasks.setRefreshing(false);
+                        }
+                    }.start();
```

Comment:
Is there a better way to avoid that problem? We don't want to intentionally give users the impression that the app is slow. If not, can we at least make it like 0.5 seconds?